src/main/java/Cart.java

```diff
@@ -0,0 +1,74 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cart {
+    private final List<CartItem> cartItems; // избрани продукти
+    private double deliveryFee;
+
+    public Cart() {
+        this.cartItems = new ArrayList<>();
+    }
+
+    double getVAT() {
```

Comment:
Seems that this method is called only internally in this class, let's change the access modifier.